wvd-templates/Create and provision WVD host pool/mainTemplate.json

```diff
@@ -292,81 +357,170 @@
                 "platformFaultDomainCount": 2
             },
             "sku": {
-                "name": "[variables('avSetSKU')]"
+                "name": "[if(variables('rdshManagedDisks'), 'Aligned', 'Classic')]"
             }
         },
         {
-            "apiVersion": "2018-05-01",
-            "name": "[variables('vmCreation-linkedTemplate_name')]",
-            "type": "Microsoft.Resources/deployments",
+            "condition": "[equals(variables('rdshImageSourceLower'), 'customvhd')]",
```

Comment:
@nakranimohit0 : Did you mean to create the image object, even if they choose "customvhd" and "rdshUseManagedDisks" = false ?

Reply:
Ah nice catch ! You are right, I need to another condition to it. I wonder how come your unmanaged custom VHD test didn't fail. It should've attempted to create this resource and fail, right ?

Reply:
It succeeded since it created the disk, it just didn't use it.